demos/horizontal_text_detection/python/rtsp_client.py

```diff
@@ -0,0 +1,133 @@
+#
+# Copyright (c) 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import cv2
+import time
+import argparse
+from functools import partial
+import tritonclient.grpc as grpcclient
+import numpy as np
+import subprocess
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--grpc_address', required=False, default='localhost', help='Specify url to grpc service')
+parser.add_argument('--grpc_port', required=False, default=9000, type=int, help='Specify port to grpc service')
+parser.add_argument('--input_stream', required=False, default="rtsp://localhost:8080/channel1", type=str, help='Url of input rtsp stream')
+parser.add_argument('--output_stream', required=False, default="rtsp://localhost:8080/channel2", type=str, help='Url of output rtsp stream')
+parser.add_argument('--fps', required=False, default=30, type=int, help='Framerate of input stream')
+args = parser.parse_args()
+output_stream = args.output_stream
+
+address = "{}:{}".format(args.grpc_address, args.grpc_port)
+triton_client = grpcclient.InferenceServerClient( url=address, verbose=False)
+
+WIDTH = 704
+HEIGHT = 704
+cap = cv2.VideoCapture(args.input_stream, cv2.CAP_FFMPEG)
+force_exit = False
+
+def decode(text):
+    word = ''
+    last_character = None
+    for character in text:
+        if character == last_character:
+            continue
+        elif character == '#':
+            last_character = None
+        else:
+            last_character = character
+            word += character
+    return word
+
+def get_text(output):
+    alphabet = '#1234567890abcdefghijklmnopqrstuvwxyz'
+    preds = output.argmax(2)
+    word = ''
+    for i in range(preds.shape[0]):
+        word += alphabet[preds[i,0]]
```

Comment:
Maybe we can rephrase that to for each i.e.
```
for pred in preds:
    word += alphabet[pred[0]]
```

Also I think adding a comment at the beginning of this function with information what kind of structure you expect as the output argument would be nice.